mitodl/edx-sga
# Fix loadjs to use window.baseUrl provided by LMS

#### What are the relevant tickets?
https://github.com/mitodl/edx-sga/issues/318

#### What's this PR do?
The XBlock fail to load when LMS static files are stored in a CDN (ex: S3)

#### How should this be manually tested?
* Use the XBlock in an LMS deployment that stores static files locally. It should work fine
* Use the XBlock in an LMS deployment that stores static files on an S3 server. It should work fine

#### Where should the reviewer start?
(Optional)

#### Any background context you want to provide?
(Optional)

#### Screenshots (if appropriate)
(Optional)

#### What GIF best describes this PR or how it makes you feel?
(Optional)
